CONTRIBUTING.md

```diff
@@ -41,3 +41,13 @@ There are various other tasks, see `./gradlew help tasks | grep liquibase`. Note
 Liquibase Hub or Liquibase Pro.
 
 DB migrations should be written to roll back cleanly (exceptions should be discussed with the team).
+
+-------------
+
+# swatch-product-configuration
+
+Directory structure: `$Platform/$Subscription_Name.yaml` (e.g. RHEL/RHEL_for_x86.yaml).
+
+Use camel case for yaml properties that need to be serialized into the `com.redhat.swatch.configuration.registry` object definitions.
```

Comment:
File naming convention came from the epic design/rationale doc.  All the Tags were listed with the expected tags and what file they should be in.

> note: underscores instead of spaces as a convention

https://docs.google.com/document/d/1qrYBxqbf9FOBKWAOYD6RDyQoj8AzNomF4cHRkAEhc9o/edit#heading=h.c52vkgk01ksn